### PR TITLE
User story 44 - Merchant deletes an item

### PR DESCRIPTION
**Pull request by tylertomlinson**
## All Submissions:

* [x] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [x] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [x] What stories does the PR include

### New Feature Submissions:

* [x] Have you added an explanation of what your changes do?
* [x] Have you lint your code locally prior to submission?

### Testing:

* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [x] All new and existing tests passed.

**Test coverage** = 100% features 100% models
--------------------------------------------------------------------------------
### Change Log

Update user edit page to remove password field
Add `profile_path` to user
Testing merchant can delete an item 
Added items resources on merchant namespace

### CODE REVIEW

File: spec/features/merchant/items_delete_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      expect(page).to have_content(item1.inventory)
+    end
+
+    click_on "Delete"
+
+    expect(current_path).to eq("/merchants/#{merchant.id}/items")
+    expect(page).to have_content("Item Successfully Deleted.")
+    expect(page).to_not have_content(item1.name)
+  end
+end
```
Comment (alex-latham) [inline spec/features/merchant/items_delete_spec.rb:29]: To fulfill all parts of US 44, we need to ensure that items can only be deleted if they have never been ordered before.

**Review by Mariana-21 — APPROVED** [review]
Clean code

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -34,12 +34,11 @@ def update_password
     user.update(user_params)
     if user.save
       flash[:success] = "Your password has been updated."
-      redirect_to '/profile'
+      redirect_to profile_path
```
Comment (danielpselph) [inline app/controllers/users_controller.rb:37]: Good use of path helpers

**Review by danielpselph — APPROVED** [review]
We should try to refactor and use path helpers throughout app

File: spec/features/merchant/items_delete_spec.rb (hunk 1 above)
Comment (alex-latham) [inline spec/features/merchant/items_delete_spec.rb:29]: Let's make sure there is no link to delete items that have been ordered before

**Pull request merged**